examples/moonshot-mainseries/molecules/filter-cdd-export.py

```diff
@@ -0,0 +1,78 @@
+#!/bin/env python
+"""
+Filter CDD export for main medicinal chemistry series
+
+https://app.collaborativedrug.com/vaults/5549/searches/11042338-xOBBXlC_s3dSQsHW3UaO2Q#search_results
+
+Header:
+Molecule Name,Canonical PostEra ID,suspected_SMILES,ProteaseAssay_Fluorescence_Dose-Response_Weizmann: IC50 (µM),ProteaseAssay_Fluorescence_Dose-Response_Weizmann: IC50 CI (Lower) (µM),ProteaseAssay_Fluorescence_Dose-Response_Weizmann: IC50 CI (Upper) (µM),ProteaseAssay_Fluorescence_Dose-Response_Weizmann: Hill slope,ProteaseAssay_Fluorescence_Dose-Response_Weizmann: Curve class
+
+"""
+
+cdd_csv_filename = 'CDD CSV Export.csv'
+output_csv_filename = 'mainseries.csv'
+
+# Load in data to pandas dataframe
+import pandas as pd
+df = pd.read_csv(cdd_csv_filename, dtype=str)
+print(f'{len(df)} records read')
+
+# Drop NaNs
+print(f'Dropping NaNs...')
+df.dropna(axis=0, how='any', thresh=None, subset=None, inplace=True)
+print(f'{len(df)} records remain')
```

Comment:
we should add an exception here in case `len(df) == 0`

Reply:
@JenkeScheen : Could you help generalize this and migrate it to https://github.com/choderalab/covid-moonshot-ml?

Reply:
IIRC @kaminow is doing this for the moonshot manuscript?

Reply:
@jchodera @JenkeScheen I've just opened a new PR for this: https://github.com/choderalab/covid-moonshot-ml/pull/130